WordPress/Sniffs/VIP/ValidatedSanitizedInputSniff.php

```diff
@@ -90,11 +90,14 @@ public function process( PHP_CodeSniffer_File $phpcsFile, $stackPtr ) {
 
 		// Handling string interpolation
 		if ( T_DOUBLE_QUOTED_STRING === $tokens[ $stackPtr ]['code'] ) {
-			foreach ( $superglobals as $superglobal ) {
-				if ( false !== strpos( $tokens[ $stackPtr ]['content'], $superglobal ) ) {
-					$phpcsFile->addError( 'Detected usage of a non-sanitized, non-validated input variable: %s', $stackPtr, null, array( $tokens[ $stackPtr ]['content'] ) );
-					return;
-				}
+			$interpolated_variables = array_map(
+				function ( $symbol ) {
```

Comment:
@olefredrik Here's the problem: I used a closure. I assumed PHP_CodeSniffer required PHP 5.3 or above, but it seems it is compatible back to 5.1. So we'll need to adjust our Travis config to test in 5.1 as well, since we test in 5.3 or above only.